index.js

```diff
@@ -27,13 +28,21 @@ const {
 
 const app = express();
 const port = PORT || 3000;
+const sess = {
+  secret: GRAVITY_SECRET,
```

Comment:
Yeah I just used the existing app secret to avoid confusion and the overhead of managing a new secret only used for cookies